frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -59,7 +59,7 @@ export const stlIsosurfaceConstants = {
 
 // This file defines the component MeshesView.
 
-const mapStateToProps = (state: OxalisState) => ({
+const mapStateToProps = (state: OxalisState): * => ({
```

Comment:
Took me a while to figure out why the errors regarding the `Props` were not caught by flow. It turns out, flow simply assumed the props to be `any`. The reason for this is that the extraction of the return type for these functions here didn't work. After a bit of digging I found out that the `*` as a return type helps :facepalm: 

Here's the comment: https://github.com/facebook/flow/issues/7071#issuecomment-553896587
The issue was opened 2.5 years ago. Yet another reason to ditch flow?

I also looked for `ExtractReturn` and its usages are very rare. So, hopefully no other bad suprises.